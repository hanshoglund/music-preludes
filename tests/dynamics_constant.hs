scat $ zipWith dynamics [fff,ff,_f,mf,mp,_p,pp,ppp] [c..]
