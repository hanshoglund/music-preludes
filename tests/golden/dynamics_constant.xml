<?xml version='1.0' ?>
<score-partwise>
  <movement-title></movement-title>
  <identification>
    <creator type="composer"></creator>
  </identification>
  <part-list>
    <score-part id="P1">
      <part-name></part-name>
    </score-part>
  </part-list>
  <part id="P1">
    <measure number="1">
      <attributes>
        <key>
          <fifths>0</fifths>
          <mode>major</mode>
        </key>
      </attributes>
      <attributes>
        <divisions>768</divisions>
      </attributes>
      <direction>
        <direction-type>
          <metronome>
            <beat-unit>quarter</beat-unit>
            <per-minute>60</per-minute>
          </metronome>
        </direction-type>
      </direction>
      <attributes>
        <time>
          <beats>4</beats>
          <beat-type>4</beat-type>
        </time>
      </attributes>
      <attributes>
        <clef>
          <sign>G</sign>
          <line>2</line>
        </clef>
      </attributes>
      <direction>
        <direction-type>
          <dynamics>
            <fff />
          </dynamics>
        </direction-type>
      </direction>
      <note>
        <pitch>
          <step>C</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>3072</duration>
        <voice>1</voice>
        <type>whole</type>
      </note>
    </measure>
    <measure number="2">
      <direction>
        <direction-type>
          <dynamics>
            <ff />
          </dynamics>
        </direction-type>
      </direction>
      <note>
        <pitch>
          <step>D</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>3072</duration>
        <voice>1</voice>
        <type>whole</type>
      </note>
    </measure>
    <measure number="3">
      <direction>
        <direction-type>
          <dynamics>
            <f />
          </dynamics>
        </direction-type>
      </direction>
      <note>
        <pitch>
          <step>E</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>3072</duration>
        <voice>1</voice>
        <type>whole</type>
      </note>
    </measure>
    <measure number="4">
      <direction>
        <direction-type>
          <dynamics>
            <mf />
          </dynamics>
        </direction-type>
      </direction>
      <note>
        <pitch>
          <step>F</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>3072</duration>
        <voice>1</voice>
        <type>whole</type>
      </note>
    </measure>
    <measure number="5">
      <direction>
        <direction-type>
          <dynamics>
            <mp />
          </dynamics>
        </direction-type>
      </direction>
      <note>
        <pitch>
          <step>G</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>3072</duration>
        <voice>1</voice>
        <type>whole</type>
      </note>
    </measure>
    <measure number="6">
      <direction>
        <direction-type>
          <dynamics>
            <p />
          </dynamics>
        </direction-type>
      </direction>
      <note>
        <pitch>
          <step>A</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>3072</duration>
        <voice>1</voice>
        <type>whole</type>
      </note>
    </measure>
    <measure number="7">
      <direction>
        <direction-type>
          <dynamics>
            <pp />
          </dynamics>
        </direction-type>
      </direction>
      <note>
        <pitch>
          <step>B</step>
          <alter>0.0</alter>
          <octave>4</octave>
        </pitch>
        <duration>3072</duration>
        <voice>1</voice>
        <type>whole</type>
      </note>
    </measure>
    <measure number="8">
      <direction>
        <direction-type>
          <dynamics>
            <ppp />
          </dynamics>
        </direction-type>
      </direction>
      <note>
        <pitch>
          <step>C</step>
          <alter>0.0</alter>
          <octave>5</octave>
        </pitch>
        <duration>3072</duration>
        <voice>1</voice>
        <type>whole</type>
      </note>
    </measure>
  </part>
</score-partwise>
